From b8a16a5178de85381339ceeea14d89f648a30f13 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Mon, 2 Oct 2017 09:04:40 +0300
Subject: [PATCH 2018/2241] defconfig: Enlarge default CMA size to 256 in
 v8_lsp_defconfig

Set the default maximum CMA size to 256MB, limited by 15% of the total
system memory.
This means that systems with less than 2GB of DRAM, will have less than
256MB of CMA memory.
The 256MB of CMA is required for a proper operation of Marvell's
user-space drivers (aka MUSDK).

Change-Id: Ifb0c1bf33a69169ec7018c354fc25ee17b5e4bea
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/44946
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index ce7a998..5b05706 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -76,7 +76,9 @@ CONFIG_UEVENT_HELPER_PATH="/sbin/hotplug"
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_DMA_CMA=y
-CONFIG_CMA_SIZE_MBYTES=64
+CONFIG_CMA_SIZE_MBYTES=256
+CONFIG_CMA_SIZE_PERCENTAGE=15
+CONFIG_CMA_SIZE_SEL_MIN=y
 CONFIG_VEXPRESS_CONFIG=y
 CONFIG_MTD=y
 CONFIG_MTD_CMDLINE_PARTS=y
-- 
2.7.4

